config/checkstyle_resources_suppressions.xml

```diff
@@ -66,4 +66,12 @@
   <suppress checks="OuterTypeFilename"
     files="src[\\/]it[\\/]resources[\\/].*[\\/]SuppressionXpathRegressionOuterTypeFilename.*\.java"/>
 
+  <!-- intentional problem for testing -->
+  <suppress checks="RegexpSingleline"
+    files="[\\/]InputJavadocContentLocationTrailingSpace\.java"/>
+  <suppress checks="RegexpSingleline"
+    files="[\\/]InputJavadocParagraphCorrect\.java"/>
+  <suppress checks="RegexpSingleline"
+    files="[\\/]InputJavadocTagContinuationIndentationGuavaFalsePositive\.java"/>
```

Comment:
You have confirmed these suppressions are needed and cannot be removed since the check and test requires the trailing whitespace for something it is testing?

Can you explain why each needs a trailing whitespace?

Reply:
I removed one which was possible, namely ``` InputJavadocParagraphCorrect.java ```.
I removed one out of the two whitespaces from ``` InputJavadocTagContinuationIndentationGuavaFalsePositive.java ```
I don't thing it's okay to remove any from ``` InputJavadocContentLocationTrailingSpace.java ``` as the check is for testing the same thing.